proxywasm/hostcall.go

```diff
@@ -18,8 +18,10 @@ import (
 	"errors"
 	"fmt"
 	"math"
+	"sync/atomic"
```

Comment:
webassembly is single-threaded (currently) and in envoy, that is the same for foreseeable future by design, so you shouldn't need atomic package

Reply:
i mean in non main pkg, so host mock and proxytest pkg are the exceptions